tests/Integration/goldens/logging/src/V2/Gapic/ConfigServiceV2GapicClient.php

```diff
@@ -78,7 +78,7 @@
  * ```
  * $configServiceV2Client = new ConfigServiceV2Client();
  * try {
- *     $formattedParent = $configServiceV2Client->organizationLocationName('[ORGANIZATION]', '[LOCATION]');
+ *     $formattedParent = $configServiceV2Client->locationName('[PROJECT]', '[LOCATION]');
```

Comment:
What is causing all these changes?

Reply:
I think it's a different version of the protos or a difference in the output of `proto_library_with_info`.